caesars-palace/blocks/carousel/carousel.css

```diff
@@ -87,8 +87,170 @@
   font-size: 20px;
 }
 
+.carousel.block .carousel-dots {
+  display: none;
+}
+
+.showcase.block .carousel-text {
```

Comment:
IS this part of the carousel block? Would `.carousel.showcase.block` be more specific?

Reply:
It is part of the carousel block. Wasn't clear to me I should also add `carousel` here, since `showcase` is at the same level. (Change done).